pytest_mypy_plugins/tests/test-regex_assertions.shouldfail.yml

```diff
@@ -0,0 +1,10 @@
+- case: rexex_but_not_turned_on
+  main: |
+    a = 'hello'
+    reveal_type(a)  # N: .*str.*
+
+- case: rexex_but_turned_off
```

Comment:
Let's add one more test that should fail: with wrong regex. Something that does not match the output 👍